### PR TITLE
Clarify where to import factory module in Phoenix

### PR DESCRIPTION
**Pull request by arsduo**
Being new to Phoenix I spent some time this morning trying to figure out why I got the error `undefined function insert/1` when setting up ExMachina. Turns out I'd put the `import MyApp.Factory` into the body of MyApp.ConnCase instead of in the `using` block. 

This PR adds a note to the readme in the hope of saving other developers the time it took to figure that out.

Thanks for the great library!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +152,8 @@ string_params_with_assocs(:comment, attrs)
 # Example of use in Phoenix with a factory that uses ExMachina.Ecto
 defmodule MyApp.MyModuleTest do
   use MyApp.ConnCase
-  # You can also import this in your MyApp.ConnCase if using Phoenix
+  # You can also import this in your MyApp.ConnCase (inside the using block)
+  # if using Phoenix
```
Comment (paulcsmith) [inline README.md]: Maybe we could shorten this to `If using Phoenix, import this inside the using block in MyApp.ConnCase`. I think that should fit on one line

**Comment by paulcsmith** [comment]
@arsduo Great idea! I had just one comment. Other than that this looks great :D

**Comment by arsduo** [comment]
Thanks for the feedback! I've updated the wording as you suggested.

**Comment by paulcsmith** [comment]
Wonderful change. Thanks for submitting it!
